webpack/extensions.ts

```diff
@@ -6,25 +6,21 @@
 
 import path from "path";
 import type webpack from "webpack";
-import * as vars from "./src/common/vars";
-import { cssModulesWebpackRule, fontsLoaderWebpackRules, iconsAndImagesWebpackRules } from "./webpack.renderer";
+import { cssModulesWebpackRule, fontsLoaderWebpackRules, iconsAndImagesWebpackRules } from "./renderer";
+import { extensionEntry, extensionOutDir, isDevelopment } from "./vars";
 
 export default function generateExtensionTypes(): webpack.Configuration {
-  const { isDevelopment } = vars;
-  const entry = "./src/extensions/extension-api.ts";
-  const outDir = "./src/extensions/npm/extensions/dist";
-
   return {
     // Compile for Electron for renderer process
     // see <https://webpack.js.org/configuration/target/>
     target: "electron-renderer",
-    entry,
+    entry: extensionEntry,
     // this is the default mode, so we should make it explicit to silence the warning
     mode: isDevelopment ? "development" : "production",
     output: {
       filename: "extension-api.js",
       // need to be an absolute path
-      path: path.resolve(__dirname, `${outDir}/src/extensions`),
+      path: path.resolve(extensionOutDir, "src", "extensions"),
```

Comment:
Is there any difference in `path.resolve("path", "sub/path")` and `path.resolve("path", "sub", "path")`?
_(I was thinking it should be covered in both cases no matter which OS/path separator)._

Reply:
I don't think so and having checked `path.resolve` will replace `/` with `\` on windows.